ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -457,6 +457,18 @@ static void setRetentionFileNum(RaftProperties properties, int numSnapshotFilesR
   interface DataStream {
     String PREFIX = RaftServerConfigKeys.PREFIX + ".data-stream";
 
+    String ASYNC_REQUEST_THREAD_POOL_CACHED_KEY = PREFIX + ".async.request.thread.pool.cached";
+    boolean ASYNC_REQUEST_THREAD_POOL_CACHED_DEFAULT = false;
+
+    static boolean useCachedThreadPool(RaftProperties properties) {
+      return getBoolean(properties::getBoolean, ASYNC_REQUEST_THREAD_POOL_CACHED_KEY,
+          ASYNC_REQUEST_THREAD_POOL_CACHED_DEFAULT, getDefaultLog());
+    }
+
+    static void setUseCachedThreadPool(RaftProperties properties, boolean useCached) {
```

Comment:
@captainzmc , thanks for the update. We also need to update the getter and setter method names. Otherwise, TestConfUtils will fail.